<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addComputer</class>
 <widget class="QDialog" name="addComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>373</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Computer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_name">
     <item>
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buildYear">
     <item>
      <widget class="QLabel" name="label_buildYear">
       <property name="text">
        <string>Build year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_buildYear">
       <property name="text">
        <string>Was not built</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorYear">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_buildYear"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_type">
     <item>
      <widget class="QLabel" name="label_type">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorType">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_type"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxConnectPers">
     <property name="text">
      <string>Connect to a Scientist</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listAllPersons">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_addComputer">
     <property name="text">
      <string>Add Computer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxConnectPers</sender>
   <signal>clicked(bool)</signal>
   <receiver>listAllPersons</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_buildYear</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_buildYear</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
